live-site/components/hacking-remaning/HackingRemaining.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+import {
+  StyledHackingRemainingContainer,
+  StyledHackingRemainingHeader,
+  StyledTime
+} from './HackingRemaining.styles';
+
+const HackingRemaining: React.FC = () => {
+  interface TimeLeft {
+    hours: number;
+    minutes: number;
+    seconds: number;
+  }
+
+  const calculateTimeLeft = (): TimeLeft => {
+    const difference = +new Date('2023-02-28T18:20:00+05:30') - +new Date();
+    let timeLeft = { hours: 0, minutes: 0, seconds: 0 };
+
+    if (difference > 0) {
+      timeLeft = {
+        hours: Math.floor(difference / (1000 * 60 * 60)),
+        minutes: Math.floor((difference / 1000 / 60) % 60),
+        seconds: Math.floor((difference / 1000) % 60)
+      };
+    }
+
+    return timeLeft;
+  };
+
+  const [timeLeft, setTimeLeft] = useState(calculateTimeLeft());
+
+  useEffect(() => {
+    setTimeout(() => {
+      setTimeLeft(calculateTimeLeft());
+    }, 1000);
+  });
+
+  return (
```

Comment:
deployment should be restricted to when hackathon starts but currently it is visible on live site if we were to push this code out

Reply:
^^ Yep we should only be displaying this once the hackathon starts so add logic to only render this then

When @surbhi-gulati and I re-review this we can just change the date locally to see it + review it again

Reply:
@jzhzhang66  what is the date and time for the hackathon start?

Reply:
Let's say Feb 10 (5pm) - Feb 12 (12pm)